packages/contracts-bedrock/src/L1/opcm/OPContractsManagerV2.sol

```diff
@@ -814,6 +885,27 @@ contract OPContractsManagerV2 is ISemver, OPContractsManagerUtilsCaller {
             )
         );
 
+        // Now we can migrate any liquidity into the ETHLockbox.
+        // The OPTIMISM_PORTAL_INTEROP dev feature is guaranteed to be enabled at this point due
+        // to the earlier check that is performed alongside the _hasInstructionByKey() check.
+        if (_hasInstructionByKey(_cfg.extraInstructions, Constants.INTEROP_MIGRATION_ADDRESSES)) {
+            // First we migrate any portal liquidity into the ETHLockbox.
+            // There may or may not be ETH in the portal, but we can call migrateLiquidity() safely
+            // either way.
+            // Authorize the portal to use the new shared ETHLockbox, then migrate the portal's liquidity into the new
+            // lockbox.
+            _cts.ethLockbox.authorizePortal(_cts.optimismPortal);
```

Comment:
This authorization is actually not necessary because ETHLockbox authorizes the portal in the ETHLockbox initializer, but it doesn't hurt either.